<details><summary>Linter check found the following problems:</summary>
Traceback (most recent call last):
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/run.py", line 208, in main
    return_code, report = execute_linter(
                          ^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/run.py", line 180, in execute_linter
    result = linter.lint(recipes, subdir, variant_config_files, exclusive_config_files, fix_flag)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 725, in lint
    msgs = self.lint_file(
           ^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 831, in lint_file
    self.lint_recipe(
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 891, in lint_recipe
    res = self.check_instances[check].run(recipe, fix)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 294, in run
    self.check_recipe(recipe)
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/check_multi_output.py", line 38, in check_recipe
    unique_names = [recipe.get("package/name")]
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/percy/render/recipe.py", line 607, in get
    raise KeyError(f"No '{path}' in Recipe {self}") from None
KeyError: "No 'package/name' in Recipe /tmp/abs_13t_hmwon1/clone/recipe"

ERROR conda.cli.main_run:execute(125): `conda run conda-lint /tmp/abs_13t_hmwon1/clone` failed. (See above for error)
</details>

--- a/recipe/LICENSE
+++ b/recipe/LICENSE
@@ -1,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2014-2018, Project Jupyter Contributors
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -2,7 +2,7 @@
 {% set version = "0.19.2" %}
 
 package:
-  #name: {{ name }}
+  name: {{ name }}-split
   version: {{ version }}
 
 source:
@@ -17,7 +17,6 @@ source:
 build:
   number: 0
   skip: true # [py<38 or s390x]
-  #script: {{ PYTHON }} -m pip install ipyleaflet --no-deps --no-build-isolation -vv
 
 requirements:
   host:
@@ -42,11 +41,6 @@ outputs:
         - hatchling
         - jupyterlab >=4,<5
         - hatch-nodejs-version >=0.3.2
-      # run:
-      #   - hatch-jupyter-builder >=0.8.1
-      #   - jupyterlab >=4,<5
-      #   - hatch-nodejs-version >=0.3.2
-        
     
   - name: {{ name }}
     version: {{ version }}
@@ -66,39 +60,18 @@ outputs:
         - jupyter_leaflet
       run_constrained:
         - jupyter_leaflet ={{ version }}
-    # test:
-    #   requires:
-    #     - python
-    #   imports:
-    #     - ipyleaflet
-
-# requirements:
-#   host:
-#     - python
-#     - pip
-#     - hatchling
-#     - setuptools
-#     - wheel
-#     - jupyter-packaging >=0.12,<1
-#   run:
-#     - python
-#     - ipywidgets >=7.6.0,<9
-#     - traittypes >=0.2.1,<0.3.0
-#     - xyzservices >=2021.8.1
-#     - branca >=0.5.0
-#     - jupyter-leaflet
-
-test:
-  imports:
-    - ipyleaflet
-  requires:
-    - pip
-    - m2-grep # [win]
-    - jupyterlab
-  commands:
-    - pip check
-    - jupyter labextension list
-    - jupyter labextension list 2>&1 | grep -ie " jupyter-leaflet.*OK"
+    test:
+      imports:
+        - ipyleaflet
+      requires:
+        - pip
+        - m2-grep # [win]
+        - jupyterlab
+        - python
+      commands:
+        - pip check
+        - jupyter labextension list
+        - jupyter labextension list 2>&1 | grep -ie " jupyter-leaflet.*OK"
 
 about:
   home: https://github.com/jupyter-widgets/ipyleaflet

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -42,16 +42,20 @@ outputs:
         - jupyterlab >=4,<5
         - hatch-nodejs-version >=0.3.2
     test:
+      imports:
+        - jupyter_leaflet
       requires:
         - python
+        - pip
       commands:
+        - pip check
         - test -f ${PREFIX}/share/jupyter/nbextensions/jupyter-leaflet/extension.js                             # [unix]
         - test -f ${PREFIX}/share/jupyter/nbextensions/jupyter-leaflet/index.js                                 # [unix]
         - test -f ${PREFIX}/share/jupyter/labextensions/jupyter-leaflet/package.json                            # [unix]
         - if not exist %PREFIX%\\share\\jupyter\\nbextensions\\jupyter-leaflet\\extension.js (exit 1)           # [win]
         - if not exist %PREFIX%\\share\\jupyter\\nbextensions\\jupyter-leaflet\\index.js (exit 1)               # [win]
         - if not exist %PREFIX%\\share\\jupyter\\labextensions\\jupyter-leaflet\\package.json (exit 1)          # [win]
-    
+
   - name: ipyleaflet
     version: {{ version }}
     build:
@@ -73,14 +77,21 @@ outputs:
       test:
         requires:
           - python
+          - pip
         imports:
           - ipyleaflet
+        commands:
+          - pip check
 about:
   home: https://github.com/jupyter-widgets/ipyleaflet
   license: MIT
   license_family: MIT
   license_file: LICENSE
   summary: A Jupyter / Leaflet bridge enabling interactive maps in the Jupyter notebook.
+  description: |
+    Ipyleaflet is a Jupyter widget for Leaflet.js , enabling interactive maps in the Jupyter notebook.
+    Every object in ipyleaflet (including the Map, TileLayers, Layers, Controls, etc.) is interactive: 
+    you can dynamically update attributes from Python or from the browser.
   doc_url: https://ipyleaflet.readthedocs.io
   dev_url: https://github.com/jupyter-widgets/ipyleaflet
 

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -41,8 +41,18 @@ outputs:
         - hatchling
         - jupyterlab >=4,<5
         - hatch-nodejs-version >=0.3.2
+    test:
+      requires:
+        - python
+      commands:
+        - test -f ${PREFIX}/share/jupyter/nbextensions/jupyter-leaflet/extension.js                             # [unix]
+        - test -f ${PREFIX}/share/jupyter/nbextensions/jupyter-leaflet/index.js                                 # [unix]
+        - test -f ${PREFIX}/share/jupyter/labextensions/jupyter-leaflet/package.json                            # [unix]
+        - if not exist %PREFIX%\\share\\jupyter\\nbextensions\\jupyter-leaflet\\extension.js (exit 1)           # [win]
+        - if not exist %PREFIX%\\share\\jupyter\\nbextensions\\jupyter-leaflet\\index.js (exit 1)               # [win]
+        - if not exist %PREFIX%\\share\\jupyter\\labextensions\\jupyter-leaflet\\package.json (exit 1)          # [win]
     
-  - name: {{ name }}
+  - name: ipyleaflet
     version: {{ version }}
     build:
       script: cd ipyleaflet && python -m pip install . --no-deps --no-build-isolation -vv
@@ -60,19 +70,11 @@ outputs:
         - jupyter_leaflet
       run_constrained:
         - jupyter_leaflet ={{ version }}
-    test:
-      imports:
-        - ipyleaflet
-      requires:
-        - pip
-        - m2-grep # [win]
-        - jupyterlab
-        - python
-      commands:
-        - pip check
-        - jupyter labextension list
-        - jupyter labextension list 2>&1 | grep -ie " jupyter-leaflet.*OK"
-
+      test:
+        requires:
+          - python
+        imports:
+          - ipyleaflet
 about:
   home: https://github.com/jupyter-widgets/ipyleaflet
   license: MIT

